build.ts

```diff
@@ -81,4 +84,9 @@ const esmBuild = () =>
 
 Promise.all([esmBuild(), cjsBuild()])
 
-exec(`tsc ${isWatch ? '-w' : ''} --emitDeclarationOnly --declaration --project tsconfig.build.json`)
+await $`tsc ${isWatch ? '-w' : ''} --emitDeclarationOnly --declaration --project tsconfig.build.json`.nothrow()
+
+console.log('Removing private fields...')
```

Comment:
Currently, this `log` shows only once.

It's okay but, it will be better show per file because it spends little long time.

https://github.com/user-attachments/assets/a1811e2b-c962-4cc4-b9dc-309c62c034f1

How do you think about it? If you agree,  remove this line and add the following inside `for await`.

```ts
console.log(`Removing private fields... ${entry}`)
```

Reply:
Yeah, it looks cool for me. I implemented it.